tests/Microsoft.Bot.Builder.AI.LUIS.Tests/TestData/EntityTextAutoSplit.json

```diff
@@ -0,0 +1,157 @@
+{
+  "entities": {
+    "$instance": {
+      "parent": [
+        {
+          "endIndex": 12,
+          "modelType": "Prebuilt Entity Extractor",
+          "recognitionSources": [
+            "model"
+          ],
+          "startIndex": 0,
+          "text": "bart simpson",
+          "type": "builtin.personName"
+        }
+      ],
```

Comment:
Was this generated by hand or by passing through LUIS?  If done by hand, it will be blown away next time someone passes through LUIS.  Checkout the comments in LuisSettings.